lib/elixir/pages/patterns-and-guards.md

```diff
@@ -382,13 +382,13 @@ Check.empty?({})
 
 In the examples above, we have used the match operator (`=`) and function clauses to showcase patterns and guards respectively. Here is the list of the built-in constructs in Elixir that support patterns and guards.
 
-  * the match operator (`=`) (exceptionally does not support guards):
+  * the match operator (`=`) intentionally does _not_ support guards:
```

Comment:
I think the word intentional is a bit weird here. I believe the word exception makes sense, because we just said that these are the constructs that support them. Alternatively we can move this particular note to the bottom.

Reply:
The disadvantage of using the word 'exceptionally' is that it makes it look like it might have to do with exceptions.

I agree that it makes sense to move this note down to the bottom.